cmd/cluster/resync.go

```diff
@@ -0,0 +1,157 @@
+package cluster
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	hiveinternalv1alpha1 "github.com/openshift/hive/apis/hiveinternal/v1alpha1"
+	"github.com/openshift/osdctl/pkg/k8s"
+	"github.com/openshift/osdctl/pkg/utils"
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	twentyMinuteTimeout   = 20 * time.Minute
+	twentySecondIncrement = 20 * time.Second
+)
+
+type Resync struct {
+	hive      client.Client
+	clusterId string
+}
+
+func newCmdResync() *cobra.Command {
+	r := Resync{}
+
+	resyncCmd := &cobra.Command{
+		Use:   "resync",
+		Short: "Force a resync of a cluster from Hive",
+		Long: `Force a resync of a cluster from Hive
+
+  Normally, clusters are periodically synced by Hive every two hours at minimum. This command deletes a cluster's
+  clustersync from its managing Hive cluster, causing the clustersync to be recreated in most circumstances and forcing
+  a resync of all SyncSets and SelectorSyncSets. The command will also wait for the clustersync to report its status
+  again (Success or Failure) before exiting.
+`,
+		Example: `
+  # Force a cluster resync
+  osdctl cluster resync --cluster-id ${CLUSTER_ID}
+`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return r.Run(context.Background())
+		},
+	}
+
+	resyncCmd.Flags().StringVarP(&r.clusterId, "cluster-id", "C", "", "OCM internal/external cluster id or cluster name to resize infra nodes for.")
```

Comment:
```suggestion
	resyncCmd.Flags().StringVarP(&r.clusterId, "cluster-id", "C", "", "OCM internal/external cluster id or cluster name to resync")
```

Reply:
Thanks for catching this! I had also the output so it prints the status condition message https://github.com/openshift/osdctl/pull/434/files#diff-069d6fe958f78782a05338fa4bfc6b1768464774df21bb4e97690bcdbde44316R145 if the clustersync isn't in a Success state per your other suggestion